Kickstarter-iOS/Views/Controllers/LoginToutViewController.swift

```diff
@@ -75,8 +75,6 @@ public final class LoginToutViewController: UIViewController, MFMailComposeViewC
   public override func viewDidLoad() {
     super.viewDidLoad()
 
-    self.navigationController?.configureTransparentNavigationBar()
```

Comment:
Removing this because it isn't needed and it is causing the an issue where the scroll view in `SetYourPasswordViewController` can be seen under the nav bar when scrolling.